Makefile

```diff
@@ -13,8 +13,8 @@ lint:
 
 
 format:
-	ruff $(ALL_LINT_PATHS) --fix
 	isort $(ISORT_PARAMS)
+	ruff $(ALL_LINT_PATHS) --fix
```

Comment:
I added the explanation into the commit message
> I did this because sometimes there was a situation when a line with imports was too long and then ruff would complain and stop formatting. While if isort runs first it formats the import breaking it into multiple lines and all works fine.

But perhaps due to the reason that you described, it's better to keep them as they were before

Reply:
As I said don't touch this. I explained why we have it as we do.